config/locales/ar.yml

```
@@ -390,6 +390,12 @@ ar:
       books: "كتب"
       lectures: "محاضرات"
       benefits: "فوائد"
+    attributes:
+      lecture:
+          kind:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)